docs/en/sql-reference/aggregate-functions/reference/argmax.md

```diff
@@ -49,3 +50,60 @@ Result:
 │ director             │
 └──────────────────────┘
 ```
+
+**Extended example**
+
+```sql
+CREATE TABLE test
+(
+    a Nullable(String),
+    b Nullable(Int64)
+)
+ENGINE = Memory AS
+SELECT *
+FROM values(('a', 1), ('b', 2), ('c', 2), (NULL, 3), (NULL, NULL), ('d', NULL));
+
+select * from test;
+┌─a────┬────b─┐
+│ a    │    1 │
+│ b    │    2 │
+│ c    │    2 │
+│ ᴺᵁᴸᴸ │    3 │
+│ ᴺᵁᴸᴸ │ ᴺᵁᴸᴸ │
+│ d    │ ᴺᵁᴸᴸ │
+└──────┴──────┘
+
+select argMax(a, b), max(b) from test;
+┌─argMax(a, b)─┬─max(b)─┐
+│ b            │      3 │ -- argMax = b because it the first not-Null value, max(b) is from another row!
+└──────────────┴────────┘
+
+select argMax(tuple(a), b) from test;
+┌─argMax(tuple(a), b)─┐
+│ (NULL)              │ -- Tuple allows to get Null value.
+└─────────────────────┘
+
+select (argMax((a, b), b) as t).1 argMaxA, t.2 argMaxB from test;
+┌─argMaxA─┬─argMaxB─┐
+│ ᴺᵁᴸᴸ    │       3 │ -- you can use Tuple and get both (all - tuple(*)) columns for the according max(b)
+└─────────┴─────────┘
+
+select argMax(a, b), max(b) from test where a is Null and b is Null;
+┌─argMax(a, b)─┬─max(b)─┐
+│ ᴺᵁᴸᴸ         │   ᴺᵁᴸᴸ │ -- Nulls are not skipped because only Null values are available
```

Comment:
```suggestion
│ ᴺᵁᴸᴸ         │   ᴺᵁᴸᴸ │ -- All aggregated rows contains at least one `NULL` value because of the filter, so all rows are skipped, therefore the result will be `NULL`
```